### ISSUE TITLE
CLI fails to parse tsconfig.json with comments

### ISSUE THREAD

**Issue by benmerckx** [justdlabs/cli#3]
Originally reported here: justdlabs/justd#244

**Describe the bug**
Just tried to init a new project that started with `bun init`. Bun creates a `tsconfig.json` file with the following contents:

````jsonc
{
  "compilerOptions": {
    // Enable latest features
    "lib": ["ESNext", "DOM"],
    "target": "ESNext",
    "module": "ESNext",
    "moduleDetection": "force",
    "jsx": "react-jsx",
    "allowJs": true,

    // Bundler mode
    "moduleResolution": "bundler",
    "allowImportingTsExtensions": true,
    "verbatimModuleSyntax": true,
    "noEmit": true,

    // Best practices
    "strict": true,
    "skipLibCheck": true,
    "noFallthroughCasesInSwitch": true,

    // Some stricter flags (disabled by default)
    "noUnusedLocals": false,
    "noUnusedParameters": false,
    "noPropertyAccessFromIndexSignature": false
  }
}
````

Running `bunx justd-cli@latest init` next produces the following error:
````
✖ Failed to parse tsconfig.json.
````

Removing the comments (to adhere to JSON) passes the cli

**To Reproduce**
1. Create tsconfig with a comment
2. Run npx justd-cli@latest init
3. See error

**Expected behavior**
Support comments in tsconfig.json

**Comment by irsyadadl** [comment]
At least right now, just remove the //

[see this](https://x.com/irsyadadl/status/1851621722501648785?s=46&t=NindQT34pY_SSnTdZL9y6w)

**Comment by irsyadadl** [comment]
It will fix on #4
